examples/reshare/src/dkg/manager.rs

```diff
@@ -622,6 +624,7 @@ where
 
         let local_outcome = Some(DealOutcome::new(
             self.signer,
+            &union(&self.namespace, OUTCOME_NAMESPACE),
```

Comment:
Applied namespace outside of types for symmetry with DKG (although this is usually applied within types)